examples/vaadin-lts-v08-simple/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
```

Comment:
Please remove the parent dependency in here and the other poms. We try to keep the examples self contained to make the required dependencies explicit. That can save a lot of time to whoever is using the example.